packages/Button/src/Button.js

```diff
@@ -160,15 +161,24 @@ const Button = React.forwardRef((props, ref) => {
   };
 
   return (
-    <span css={buttonStyles} data-pka-anchor="button" {...buttonProps} as={isSemantic ? "button" : RawButton}>
-      <ButtonIcon {...iconProps} isPending={isPending} data-pka-anchor="button.icon">
-        {isPending ? <RefreshIcon /> : icon}
-      </ButtonIcon>
-      {children}
-      <ButtonIcon {...iconProps} isDropdown data-pka-anchor="button.icon">
-        {isDropdown && <DownIcon />}
-      </ButtonIcon>
-    </span>
+    <React.Fragment>
+      <GlobalStyle />
+      <span
+        css={buttonStyles}
+        data-pka-anchor="button"
+        {...buttonProps}
+        data-pka-type="button"
```

Comment:
Why not use `data-pka-anchor`?

Reply:
Because the consumer can pass it in and change it, like this: https://github.com/acl-services/paprika/blob/master/packages/Button/stories/examples/Basic.js#L25